index.bs

```diff
@@ -1031,6 +979,6 @@ Alvestrand, Stefan H&aring;kansson, Bernard Aboba and Dominique
 Haza&euml;l-Massieux, for their support. Contributions to this specification
 were provided by Robin Raymond.
 
-The {{QuicTransport}} and `QuicStream` objects were initially described in the
+The `QuicTransport` and `QuicStream` objects were initially described in the
```

Comment:
How about something like

> The {{WebTransport}} interface is based on the `QuicTransport` interface initially described ...

Since we're not using `QuicStream` it's probably unnecessary to mention it.